docs/topics/crud-mission-integration-testing.adoc

```diff
@@ -1,8 +1,22 @@
 [[crud-integration-testing]]
 = Running Integration Tests
 
-This booster contains a set of integration tests.
-To run them, you must be connected to an OpenShift instance and select the project that contains your CRUD application and database.
+This booster includes a set of integration tests.
+This set of tests is self-contained, meaning that if you run them inside an empty project, they
```

Comment:
The word "project" can have multiple meanings in this context; I'd suggest saying "inside an empty OpenShift project" explicitly.

Reply:
If you're doing a list mid-sentence, you should probably add a colon after they.